py/picca/delta_extraction/expected_fluxes/dr16_expected_flux.py

```diff
@@ -903,31 +932,27 @@ def populate_los_ids(self, forests):
                 continue
             # get the variance functions and statistics
             stack_delta = self.get_stack_delta(forest.log_lambda)
-            var_lss = self.get_var_lss(forest.log_lambda)
             eta = self.get_eta(forest.log_lambda)
-            fudge = self.get_fudge(forest.log_lambda)
 
             mean_expected_flux = forest.continuum * stack_delta
-            var_pipe = 1. / forest.ivar / mean_expected_flux**2
-            variance = eta * var_pipe + var_lss + fudge / var_pipe
-            weights = 1. / variance
-
+            weights = self.get_continuum_weights(forest, mean_expected_flux)
+            # this is needed as the weights from get_continuum_weights are
+            # divided by the continuum model squared, in this case mean_expected_flux
+            # TODO: check that we indeed need this or if the weights without it
```

Comment:
Ok, so this does give the same result as before, right? It might be reasonable to have some kind of Fbar weights as an alternative, I guess... But this should be optional, not replacing the current setup

Reply:
While that would be nice, depending on the type of downstream analysis that can give unexpected effects. For Pk1d for instance, using the standard weights would be relatively simple to do when estimating the power spectrum from real space data, but when doing straight-forward FFTs each pixel of a spectrum will have the same weight in the output P(k), i.e. we cannot account for the different weighting of the pixels in a straightforward way.
This does not mean we couldn't in principle do the cont fitting with some weighting scheme, but effects this has need to be studied.
Agreed that within a run, i.e. after specifying which kind of weights one wants, things should be treated self-consistently.

Reply:
Maybe we should open an issue to discuss how to proceed with this

Reply:
I think it's fine as long as we keep the current major modes (i.e. standard or constant weights, not sure if anyone uses ivar-only weights) in working shape which allows us to do analyses similar to previous works.

We could have a discussion if other weightings might be useful, then implement those, and then if they have been tested and lead to improvement I'd be fine with dropping previous modes. In principle, already the change in binning from log to lin can be seen as a λ-dependent change in weighting, at least for e.g. constant weights.